.circleci/config.yml

```
@@ -0,0 +1,155 @@
+    steps:
+      - run:
+          name: Send Slack notification
+          command: |
+            if [ "<< parameters.status >>" = "success" ]; then
+              color="good"
+              emoji=":white_check_mark:"
+            else
+              color="danger"
+              emoji=":x:"
+            fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)